packages/strapi-plugin-content-manager/admin/src/components/InputJSONWithErrors/index.js

```diff
@@ -56,7 +57,9 @@ class InputJSONWithErrors extends React.Component {
               <Label htmlFor={name}>
                 <span>{label}</span>
                 {labelIcon && (
-                  <LabelIconWrapper title={labelIcon.title}>{labelIcon.icon}</LabelIconWrapper>
+                  <FormattedMessage {...labelIcon.title}>
+                    {msg => <LabelIconWrapper title={msg}>{labelIcon.icon}</LabelIconWrapper>}
```

Comment:
Why don't we use the formatMessage function from useIntl ?

Reply:
Because this component is a class and I did want to migrate it to a functional components as I think that it should not be done in this PR but when we start the v4.